product/dialogs/service_dialogs/transform-vm.yml

```diff
@@ -0,0 +1,221 @@
+---
+- description: Transform VM
+  buttons: submit,cancel
+  label: Transform VM
+  blueprint_id:
+  dialog_tabs:
+  - description:
+    display: edit
+    label: New tab
+    display_method:
+    display_method_options:
+    position: 0
+    dialog_groups:
+    - description: Import given virtual machine to another infrastructure provider
+      display: edit
+      label: Transform Virtual Machine
+      display_method:
+      display_method_options:
+      position: 0
+      dialog_fields:
+      - name: name
+        description: Name of the created virtual machine
```

Comment:
@matobet 
Is this an existing name or the name when it its moved to the new provider?
If its an existing name do we have to worry about users typing errors, when would we validate those?